core/packages/contracts/contracts/NativeTokens.sol

```diff
@@ -0,0 +1,148 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity ^0.8.9;
+
+import "@openzeppelin/contracts/access/Ownable.sol";
+import "@openzeppelin/contracts/token/ERC20/extensions/IERC20Metadata.sol";
+
+import "./utils/MessageProtocol.sol";
+import "./ERC20Vault.sol";
+import "./OutboundChannel.sol";
+
+/// @title Native Tokens
+/// @notice A contract for managing ethereum native tokens.
+/// @dev Manages locking, unlocking ERC20 tokens in the vault. Initializes ethereum native tokens on the substrate side via create.
+contract NativeTokens is Ownable {
+    /// @notice Unlock payload format.
```

Comment:
Rather just use `@dev` instead of both `@dev` and `@notice`. We don't need to be more specific.